packages/react-core/src/components/MultipleFileUpload/MultipleFileUploadButton.tsx

```diff
@@ -21,7 +21,7 @@ export const MultipleFileUploadButton: React.FunctionComponent<MultipleFileUploa
   return (
     <div className={css(styles.multipleFileUploadUpload, className)} {...props}>
       <Button variant="secondary" aria-label={ariaLabel} onClick={open}>
-        Upload
+        Browse
```

Comment:
We need to add a prop for this so the text can be changed by the consumer and not hard coded.

Reply:
Should I set its default value to 'Browse' if no label has been provided?